tests/unit/nupic/algorithms/cla_classifier_test.py

```diff
@@ -28,7 +28,9 @@
 import unittest2 as unittest
 
 import numpy
+import tempfile
 
+from nupic.bindings.proto import ClaClassifier_capnp
```

Comment:
Seems right. Did you do a rebuild of nupic? And if you have it set up to use a local nupic.core, did you rebuild the latest from that first?